scripts/temp.deployment.sh

```diff
@@ -21,24 +22,50 @@
 # ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 # OTHER DEALINGS IN THE SOFTWARE.
 #
-# Temporary script to deploy argo onto a system for testing
-#     NOTE: this will be removed once we have deployment work done
+set -e
 
-# csm 1.2
+function deployNLS() {
+    BUILDDIR="/tmp/build"
+    mkdir -p "$BUILDDIR"
+    kubectl get secrets -n loftsman site-init -o jsonpath='{.data.customizations\.yaml}' | base64 -d > "${BUILDDIR}/customizations.yaml"
+    kubectl get configmap -n loftsman loftsman-platform -o jsonpath='{.data.manifest\.yaml}' > "${BUILDDIR}/iuf.yaml"
+    manifestgen -i "${BUILDDIR}/iuf.yaml" -c "${BUILDDIR}/customizations.yaml" -o "${BUILDDIR}/platform.yaml"
+    yq w -i "${BUILDDIR}/platform.yaml" 'spec.charts[0].version' "$2"
+    charts="$(yq r /tmp/build/platform.yaml 'spec.charts[*].name')"
```

Comment:
I have not tested this suggestion, but I think this `/tmp/build` should be using `BUILDDIR` instead.

```suggestion
    charts="$(yq r "${BUILDDIR}/platform.yaml" 'spec.charts[*].name')"
```